src/Uno.UWP/UI/Input/ManipulationDelta.cs

```diff
@@ -12,5 +25,24 @@ public partial struct ManipulationDelta
 		/// <inheritdoc />
 		public override string ToString()
 			=> $"x:{Translation.X:N0};y:{Translation.Y:N0};θ:{Rotation:F2};s:{Scale:F2};e:{Expansion:F2}";
+
+		internal ManipulationDelta Add(ManipulationDelta right) => Add(this, right);
+		internal static ManipulationDelta Add(ManipulationDelta left, ManipulationDelta right)
+			=> new ManipulationDelta
+			{
+				Translation = new Point(
+					left.Translation.X + right.Translation.X,
+					left.Translation.Y + right.Translation.Y),
+				Rotation = left.Rotation + right.Rotation,
+				Scale = left.Scale * right.Scale,
+				Expansion = left.Expansion + right.Expansion
+			};
+
+		// Note: We should apply a velocity factor to thresholds to determine if isSignificant
```

Comment:
From when I played in gestures for SwipableItems, the velocity was really key to have a fluid experience. Can't tell now without trying the PR but are we confident we don't need to consider velocity now? How much effort would it be?

Reply:
I agree that velocity is really important, but it really depends to what you do: if you want to do precise manipulation (like dragging a GridSplitter) you don't have to deal with velocity, but for a SwipableItem or even image manipulation it's a must.

I didn't checked as it was not mandatory for now, I think it would be easy on iOS as every thing is already baked-in, but for Android and WASM we will have to compute every thing by ourselves.